File: backend/back-pasteleria/src/main/java/cl/milsabores/pasteleria/dto/RegisterRequest.java
package cl.milsabores.pasteleria.dto;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;
import lombok.Data;

@Data
public class RegisterRequest {

    @Email
    @NotBlank
    private String email;

    @NotBlank
    private String password;

    @NotBlank
    private String firstName;

    @NotBlank
    private String lastName;

    @NotBlank
    private String phone;

    private String run;
    private String street;
    private String region;
    private String comuna;
    private String birthDate;
}

File: app/src/main/java/com/example/pasteleriamilssaboresandroid/data/network/dto/RegisterRequest.kt
package com.example.pasteleriamilssaboresandroid.data.network.dto

data class RegisterRequest(
    val email: String,
    val password: String,
    val firstName: String,
    val lastName: String,
    val phone: String,
    val run: String?,
    val street: String?,
    val region: String?,
    val comuna: String?,
    val birthDate: String?
)

